packages/pymedphys_dicom/src/pymedphys_dicom/dicom/anonymise.py

```diff
@@ -528,15 +539,15 @@ def unknown_tags_in_dicom_dataset(ds):
     exist in the PyMedPhys copy of the DICOM dictionary.
     """
 
-    non_private_tags_in_dataset = np.array(
-        non_private_tags_in_dicom_dataset(ds))
+    non_private_tags_in_dataset = np.array(non_private_tags_in_dicom_dataset(ds))
 
     are_non_private_tags_in_dict_baseline = [
-        tag in BaselineDicomDictionary.keys()
-        for tag in non_private_tags_in_dataset]
+        tag in BaselineDicomDictionary.keys() for tag in non_private_tags_in_dataset
+    ]
 
-    unknown_tags = list(non_private_tags_in_dataset[
-        np.invert(are_non_private_tags_in_dict_baseline)])
+    unknown_tags = list(
+        non_private_tags_in_dataset[np.invert(are_non_private_tags_in_dict_baseline)]
+    )
```

Comment:
To minimise reviewing pain I think we should do a once off pull request which blackifies the whole code base...

But hard to see what's changed through all the noise when black does its thing and there is a needle somewhere in the haystack that has also changed.

Reply:
... wow thanks @Matthew-Jennings for blackifying everything. That was certainly worth it. The diffs are all signal and no noise. It's a breath of fresh air. There's no diff due to a command or something random, every diff line highlighted is one that is relevant for review.

It's beautiful. We made a good choice. Thanks @Matthew-Jennings for doing that work. :).